README.md

```diff
@@ -0,0 +1,11 @@
+# Kitchen Sink
+
+### One timers
+
+- npm i -g yarn
+- yarn config set workspaces-experimental true
+
+### Install
+
+- git clone
+- lerna bootstrap
```

Comment:
`lerna bootstrap` did not install any of the root-level dependencies for me.

Reply:
Should this be changed to `yarn` which runs lerna bootstrap and grunt babel & sass automatically?

Reply:
Nevermind. I was on a different branch. I had a local branch with the same name as this one. Please disregard.